pest-ion/src/bin/pest2ion/main.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates.
+
+use anyhow::Result;
+use clap::{crate_authors, crate_description, crate_version, App, Arg, ArgGroup};
+use ion_rs::value::writer::{ElementWriter, Format, TextKind};
+use pest_ion::{from_read, TryPestToElement};
+use std::fs::File;
+use std::io::{stdin, stdout, Write};
+use std::path::Path;
+
+fn main() -> Result<()> {
+    let matches = App::new("Pest to Ion Converter")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .arg(
+            Arg::with_name("FILE")
```

Comment:
`INPUT_FILE`/`OUTPUT_FILE` would have a nicer symmetry IMO.

Reply:
That is reasonable, I will change this.